applications/liberland.md

```diff
@@ -0,0 +1,345 @@
+
+Direct democracy is our basic check-and-balance: the people can second-guess their representatives in Congress. In Liberland, this is used to strike down unwanted regulations but can be easily adapted for positive democracy or even as the basis for elections. We complement this system with a front end.   
+
+The hitherto unseen benefit is that voting on-chain can stay permanent. An online system's relatively low transaction costs allow a polity to use higher thresholds for adopting a motion. "Paper" referenda as implemented in current States usually rely on artificially low quora, breaking the original assumption of a "majority rule". We hope to be able to unleash the true power of the people by giving them the flexibility and the security of a public blockchain. 
+
+
+#### Executive Power    
+As required by the standards of modern Statehood, the Executive (vested in a central organ, the Cabinet) is voted in and removed by Congress. The standard of a "fully responsible" government requires States to adopt such a system or face criticism as "not fully democratic".    
+Our Executive is a fork of modern parachain councils such as Polkadot's Council, though heavily modified. It is composed of four ministers led by one prime minister. Other adoptees can expand the system, designed to handle complex administrative processes such as regulations, making decisions and even dispute resolution.
+
+
+Connection to extrinsic systems, standard or customized per-state via API will enable the system to integrate seamlessly into an existing State information system. We build e-government for Liberland, partly inspired by Estonia but fully Web-3 based.
+The powers of the Executive in Liberland are to govern directly on-chain within the limits of Congress's laws and off-chain as well in the metaverse and later the real-life state.
+    
+    
+More complex State actors might create specialized parachains which would govern their municipalities or even byzantine central bureaucracies. Given that States generate traffic as a matter of course, such systems are very interesting for Validators and Nominators and virtually non-existent in the current space.
+ 
+
+
+#### Treasury   
+The Treasury for LLM, unlike that of Polkadot and like treasuries of extant States, operates without the burn mechanism. LLD follows the same rules as Polkadot, and so there is a burn mechanism to offset the “infinite” (really: algorithmic) supply. 
+As the LLM tokenomics suppose a deflationary token, holding LLM in the treasury will help finance running the State. The System distributes a significant portion of the treasury among the Members of Congress. Through this, the tokens get directly into the hands of the Voters (Citizens). Using Pallet-llm the treasury is the owner and manager of several on-chain assets, allowing us to expand the base functionality to have the treasury securely manage on-chain assets for us. Decentralizing trust in a true way.
```

Comment:
1. Genesis: pre-mint the LLM
2. Place it in the Reserve Fund Wallet as locked
3. Have an algorithm release X amount per block into the TREASURY
4. Have the Senate hold the wallet as MultiSig
5. Have the entire Senate in consensus as requirement to do something else with the Reserve Fund than watch the algorithm do the controlled per-block release.

Treasury:
1. Have the Senate hold the deciding power to release funds from the Treasury (not consensus)
2. Have the Congress accept Spending Proposals instead of the Council ~Treasury Law
3. Have approx one Treasury Law per year which specifies which funds go to which wallet; wallets here belong to the branches of the government;
4. Upon the Spending Proposal being accepted, have it go to the Senate
5. If the Senate doesn't release the Funds, the payment doesn't go through (but the Spending Proposal funds get returned to the Wallet of the proposing account)
6. Have the

Reply:
The treasury manage on-chain assets, this a customized implementation for liberland's chain with the LLM Pallet